cluster/gce/config-default.sh

```diff
@@ -260,6 +256,18 @@ if [[ ! -z "${NODE_ACCELERATORS}" ]]; then
     fi
 fi
 
+if [[ "${KUBE_FEATURE_GATES:-}" == "AllAlpha=true" ]]; then
+  RUNTIME_CONFIG="${KUBE_RUNTIME_CONFIG:-api/all=true}"
+  if ! [[ "${KUBE_FEATURE_GATES:-}" =~ "CSIDriverRegistry" ]]; then
```

Comment:
Doesn't line 259 guarantee that KUBE_FEATURES_GATES is `ALLAlpha=true`?

I guest this block shouldn't be nested.

Reply:
Yes, but if a user explicitly disables this feature, I don't want to force it on even if it is enabled by default for alpha.